recipe/patches/0002-disable-new-dtags.patch

```diff
@@ -0,0 +1,12 @@
+--- postgresql-16.0.orig/src/makefiles/Makefile.linux	2023-09-14 10:35:53.599777660 -0500
```

Comment:
We should keep the ownership information for the patch.

Reply:
Looks like it was recreated rather than manually adjusted to account for it no longer applying to this file.
https://github.com/postgres/postgres/blob/REL_17_0/src/makefiles/Makefile.linux

`DLSUFFIX = .so` was removed but was a part of the anchoring on the previous patch.

Reply:
I would highly recommend to describe/explain where you got the patches from in the PR description.